tests/helpers/start-app.js

```diff
@@ -1,30 +1,18 @@
-/* global require */
-
-var Application = require('myapp/app')['default'];
-var Router = require('myapp/router')['default'];
 import Ember from 'ember';
+import Application from '../../app';
+import config from '../../config/environment';
 
 export default function startApp(attrs) {
-  var App;
-
-  var attributes = Ember.merge({
-    // useful Test defaults
-    rootElement: '#ember-testing',
-    LOG_ACTIVE_GENERATION:false,
-    LOG_VIEW_LOOKUPS: false
-  }, attrs); // but you can override;
+  let application;
```

Comment:
'let' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).